### DISCUSSION TITLE
Improving webpage interface

### DISCUSSION THREAD

**Discussion by camilo-zuluaga** [Sriparno08/Openpedia#90] [Ideas]
Hello @Sriparno08,

I hope you are well, I find your project very interesting, that's why I wanted to make a simple input and chat about these changes.
The website as they say, is still under construction, however, at this early stage I have seen some things that don't quite add up in the design and colors of the website.

1. As a first part, I have seen some colors that stand out a lot on the web page and do not have much relation to each other, such as the **"Get Started"** and **"More"** button, a purple color that does not follow the color standards on the page and a quite hard and bright blue, so my contribution would be that this button has the main color that is carried on the web page, *Preview:*
![Colors](https://github.com/Sriparno08/Openpedia/assets/103959811/ef3c2b0d-cd88-4836-ae13-ac8020232393)

2. As second and last part, I would like to talk about the alignment of the text in the last section, as you can see in the source code, you have a `text-align: justify;`, and this in the future **could create long blocks of text and make them difficult to read**, also you see some spaces that are quite strange to the human eye, and it is much more noticeable in the view of mobile devices, there are several articles that talk about this, so I wanted to suggest align it better to the left, *Preview*:
![text-align](https://github.com/Sriparno08/Openpedia/assets/103959811/250e2434-13c8-4e9a-990f-9eac41e12b6c)
![text-align-mobile](https://github.com/Sriparno08/Openpedia/assets/103959811/3f846b70-40fc-4aef-8183-2c5b0278b761)

The article, *[Why you shouldn't justify text on a website or app](https://ruitina.com/dont-justify-text-on-websites-and-apps/)*.

Thank you for taking the time to read it, I wish you good luck, and hopefully I can contribute to this project.

Best regards,
Camilo Zuluaga

**Comment by Sriparno08** [comment]
Hey @camilo-zuluaga, sorry for the late reply as I've just discovered this discussion.

Thanks for your interest in this project and suggesting ideas in such a detailed manner for improving the website.

In both cases, I agree with your suggestion and thus have decided to implement them to the project.

So, let me open separate issues for both of your suggestions so that you can comment on them and get assigned.

Thanks!
